### PR TITLE
Update build root

### PR DESCRIPTION
**Pull request by danilo-gemoli**
/hold
merge https://github.com/openshift/release/pull/53518 first

### CODE REVIEW

File: .ci-operator.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 build_root_image:
   namespace: ci
   name: ci-tools-build-root
-  tag: "1.21"
+  tag: "1.22"
```
Comment (droslean) [inline .ci-operator.yaml:4]: This doesn't exist.

**Comment by danilo-gemoli** [comment]
/close
superseded by https://github.com/openshift/ci-tools/pull/4203

**Pull request closed without merging**